docs/current_docs/partials/_install-cli.mdx

```diff
@@ -146,3 +148,17 @@ where.exe dagger
 
 </TabItem>
 </Tabs>
+
+### Development version
+
+:::warning
+Development versions should be considered unstable and may produce unexpected results.
+:::
+
+To install the latest development version, use the following command:
+
+```shell
+curl -fsSL https://dl.dagger.io/dagger/install.sh | DAGGER_COMMIT=head sh
+```
+
+This will replace your current Dagger CLI version, so you may prefer to install it in an isolated environment so as to avoid conflicts.
```

Comment:
_This will install the development Dagger CLI in `./bin/dagger`_

I would add here that running this CLI against the stable Engine in Docker will by default stop & remove this stable version, and install a development version of the Engine in Docker.

Reply:
Done